test/functional/JLM_Tests/src/org/openj9/test/java/lang/management/TestThreadMXBean.java

```diff
@@ -350,16 +354,70 @@ public final void testGetThreadCpuTime() {
 					AssertJUnit.assertTrue(e instanceof IllegalArgumentException);
 				}
 
+				// test getThreadCpuTime(long[])
+				try {
+					tb.getThreadCpuTime(null);
+					Assert.fail("Should have thrown NullPointerException");
+				} catch (NullPointerException e) {
+					// expected
+				}
+				try {
+					long[] badIds = new long[] {-1};
+					tb.getThreadCpuTime(badIds);
+					Assert.fail("Should have thrown IllegalArgumentException");
+				} catch (IllegalArgumentException e) {
+					// expected
+				}
+				Object sync = new Object();
+				Thread t1 = new Thread(new Runnable() {
+					@Override
+					public void run() {
+						try  {
+							Thread.sleep(100);
+							synchronized (sync) {
+								sync.notify();
+								sync.wait();
+							}
+						} catch (InterruptedException | IllegalMonitorStateException e) {
```

Comment:
I don't think `IllegalMonitorStateException` should be ignored; I don't think it is possible in this `try` block, nor any other place where it is caught in this file.

Reply:
Updated.